src/App/App.js

```diff
@@ -4,10 +4,75 @@ import Favorites from '../Favorites/Favorites.js'
 import Form from '../Form/Form.js'
 import Home from '../Home/Home.js'
 import Nav from '../Nav/Nav.js'
+import Poster from '../Poster/Poster.js'
+import { mockQuotes, mockImages, mockPosters } from '../mockData'
+import {useState, useEffect } from 'react'
+import { getData } from '../apiCalls'
 
 const App = () => {
+  const [quotes, setQuotes] = useState(mockQuotes.quotes)
+  const [images, setImages] = useState(mockImages.images)
+
+  //// --> hopefully ready for real data - needs testing/debugging
```

Comment:
So excited that these are here! I am hoping once the be is up and running it is smooth sailing!